frontend/shared/src/components/forms/section/FormSection.tsx

```diff
@@ -1,12 +1,24 @@
+import { LoadingSpinner } from 'hds-react';
 import * as React from 'react';
 
 import { StyledContent, StyledHeader, StyledSection } from './styled';
 
-type FormSectionProps = { children: React.ReactNode; header?: string };
+type FormSectionProps = {
+  children: React.ReactNode;
+  header?: string;
+  loading?: boolean;
+};
 
-const FormSection: React.FC<FormSectionProps> = ({ children, header }) => (
+const FormSection: React.FC<FormSectionProps> = ({
+  children,
+  header,
+  loading,
+}) => (
   <StyledSection>
-    <StyledHeader>{header}</StyledHeader>
+    <StyledHeader>
+      {header}
+      {loading && <LoadingSpinner small />}
```

Comment:
This is great addition as the `LoadingSpinner` from hds handles also the accessibility and screen reader will tell if the loading is ongoing. One small thing is that you can give `loadingText` and `loadingFinishedText` with translations so it will be a little bit better for screenreader
```
{loading && (
<LoadingSpinner small 
loadingText={t('common:loading')} 
loadingFinishedText={t('common:loadingFinished')} />
)}
```